src/AppInstallerCLICore/ExecutionArgs.h

```diff
@@ -46,6 +46,7 @@ namespace AppInstaller::CLI::Execution
             AcceptPackageAgreements, // Accept all license agreements for packages
             Rename, // Renames the file of the executable. Only applies to the portable installerType
             NoUpgrade, // Install flow should not try to convert to upgrade flow upon finding existing installed version
+            IgnoreMissingDependencies, // Ignores missing Windows Feature dependencies.

```

Comment:
Also, should `--force` apply here as well?

Reply:
Since you are using `--force` for overriding a failed install, I think that equally applies to one that doesn't exist.  I would remove this option.

Along those lines, I wouldn't treat an unknown feature as a special failure checked up front, but rather as a special failure when it is being satisfied in the normal flow.